releases/v0.17.0.md

```diff
@@ -0,0 +1,90 @@
+# Release Notes for v0.17.0
+
+Release Date: November 21st, 2025
+
+This release is based on [3.23.0](https://github.com/kata-containers/kata-containers/releases/tag/3.23.0) of Kata Containers
+and [v0.11.0](https://github.com/confidential-containers/enclave-cc/releases/tag/v0.11.0) of enclave-cc.
+
+Trustee and the guest components use KBS protocol v0.4.0.
+
+Please see the [quickstart guide](https://confidentialcontainers.org/docs/getting-started/) or [project documentation](https://confidentialcontainers.org/docs) for more information.
+
+## Deprecation Notices
+* Support for process-based confidential computing via enclave-cc will be removed in the v0.18.0 release.
+  The enclave-cc project will be archived.
+* The community is working on a helm-based replacement for the CoCo Operator. This will likely be
+  the last release with the operator. If you have any key requirements for the operator replacement
+  please share them with the community on GitHub or Slack. 
+
+## What's New
+
+* Improvements to PKCS11 support in Trustee.
+* Docker Compose Trustee checks attestation token signatures by default.
+* Trustee CLI too introduced. More features are in development.
+* Trustee can verify certain NVIDIA GPUs via NRAS.
+* Trustee has basic support for validating TPMs.
+* Trustee admin backend has been improved to support multiple admin keypairs.
+* The simple attestation token was removed per the deprecation process.
+* Improvements to logging in the Attestation Service.
+* Individual reference values are provided to the AS Policy via a Rego extension
+  rather than via one large map.
+* If CDH.toml or AA.toml are provided via init-data, AS will parse these files
+  as JSON to expose them to the policy and include them in the token.
+* InitData plaintext can be provided via KBS Client and Trustee Attester.
+* Astral-Tokio-Tar updated to avoid CVE.
+* Image-rs can handle images with absolute path hard links.
+
+## Hardware Support
+
+Attestation is supported and tested on three platforms: Intel TDX, AMD SEV-SNP, and IBM SE.
+Not all features have been tested on every platform, but those based on attestation
+are expected to work on the platforms above.
+
+Make sure your host platform is compatible with the hypervisor and guest kernel
+provisioned by CoCo.
+
+This release has been tested on the following stacks:
+
+### AMD SEV-SNP
+
+* Processor: AMD EPYC 7413
+* Kernel: 6.16.1
+* OS: Ubuntu 22.04.4 LTS
+* k8s: v1.33.0 (Kubeadm)
+* Kustomize: v5.6.0
+
+### Intel TDX
+
+* Kernel: [6.8.0-1028-intel](https://git.launchpad.net/~kobuk-team/ubuntu/+source/linux-intel/tree/?h=noble-main-next)
+* OS: Ubuntu 24.04 LTS
+* k8s: v1.30.2 (Kubeadm)
```

Comment:
I may have forgotten to keep this up-to-date. Just checked that we're already on:
```suggestion
* k8s: v1.34.2 (Kubeadm)
```